packages/console/src/pages/EnterpriseSsoDetails/Experience/index.tsx

```diff
@@ -113,7 +113,10 @@ function Experience({ data, isDeleted, onUpdated }: Props) {
         const updatedSsoConnector = await api
           // Only keep non-empty values since PATCH operation performs a merge scheme.
           .patch(`api/sso-connectors/${data.id}`, {
-            json: cleanDeep(formDataToSsoConnectorParser(formData)),
+            json: cleanDeep(formDataToSsoConnectorParser(formData), {
```

Comment:
When set emptyObjects and undefinedValues to false, what is the expected behavior for the cleanDeep method? P.S should we have some kind of integration test of unit test for this?

Reply:
Keeping `undefined` values can help to delete some fields of DB and keeping empty objects can help to clear JSONB fields.

Reply:
Updated, the `PATCH /sso-connectors/:id` API, we need to keep empty objects when applying `cleanDeep` method to reset the `branding` config since it is a JSONB-typed column in DB.